Casks/d/dintch.rb

```diff
@@ -9,12 +9,12 @@
   homepage "https://eclecticlight.co/dintch"
 
   livecheck do
-    url "https://raw.githubusercontent.com/hoakleyelc/updates/master/eclecticapps.plist"
-    strategy :page_match do |page|
-      match = page.match(%r{(\d+)/(\d+)/dintch(\d+)\.zip}i)
-      next if match.blank?
-
-      "#{match[3].split("", 2).join(".")},#{match[1]}.#{match[2]}"
+    url :homepage
+    regex(%r{href=.*?/(\d+)/(\d+)/dintch(\d+)\.zip}i)
+    strategy :page_match do |page, regex|
+      page.scan(regex).map do |match|
+        "#{match[2].split("", 2).join(".")},#{match[0]}.#{match[1]}"
```

Comment:
```suggestion
    url "https://eclecticlight.co/downloads/"
    regex(%r{href=.*?/(\d+)/(\d+)/dintch[^"' >]*?\.zip[^>]*?>\s*Dintch\s+v?(\d+(?:\.\d+)*)[^a-z)]}i)
    strategy :page_match do |page, regex|
      page.scan(regex).map { |match| "#{match[2]},#{match[0]}.#{match[1]}" }
```

It looks like we can check the [Downloads](https://eclecticlight.co/downloads/) page for these casks, unless there's a particular benefit to checking the `homepage`.

The downloads page also provides the version with dots in the `a` element's inner text, so we're better off matching that instead of naively trying to insert dots into the [dotless] filename version. Some of the software already has a two digit major or minor, so we can't safely make assumptions about where to insert dots.